Gemfile

```diff
@@ -41,7 +41,7 @@ gem "sdoc", group: :doc
 # Use Rails Html Sanitizer for HTML sanitization
 gem "rails-html-sanitizer"
 
-gem "react_on_rails", "~> 5.1.1"
+gem "react_on_rails", "6.0.0.rc.5"
```

Comment:
UPGRADE NOTE: This is the relevant gem change. Adjust to the latest version of React on Rails.